src/index.js

```diff
@@ -0,0 +1,44 @@
+
+require('esn-frontend-common-libs/src/index.js');
```

Comment:
Same comment here. If we really need esn commons, we need to tell webpack to not bundle it in this project